dev/run-tests-jenkins.py

```diff
@@ -181,8 +181,8 @@ def main():
     short_commit_hash = ghprb_actual_commit[0:7]
 
     # format: http://linux.die.net/man/1/timeout
-    # must be less than the timeout configured on Jenkins (currently 300m)
-    tests_timeout = "250m"
+    # must be less than the timeout configured on Jenkins (currently 350m)
+    tests_timeout = "300m"
```

Comment:
Ah, this was the root causes. Thanks, @HyukjinKwon .
Sorry, @shaneknapp . I sent an annoying email before knowing this here.